src/main/java/com/exadel/discountwebapp/baseclasses/BaseEntityMapper.java

```diff
@@ -0,0 +1,5 @@
+package com.exadel.discountwebapp.baseclasses;
```

Comment:
let's rename "baseclasses" package to "common"

Reply:
done